spec/classes/puppet_init_spec.rb

```diff
@@ -60,13 +62,15 @@
         install_hc_gem: false
       }
     end
+
     it { is_expected.not_to contain_package('hipchat') }
   end
 
   describe 'on pe' do
```

Comment:
Since the ```is_pe``` fact is no longer used, should we just rip it out altogether?

Reply:
Sure. Updated.